source/templates/application_template.html.md.erb

```diff
@@ -55,31 +64,36 @@ Do you support this application? Help out by [adding your team name][app-yaml] t
 </ul>
 
 <h3>SSH access</h3>
+<% if application.datagovuk_app? %>
```

Comment:
Could we rename the existing helper here and get it to make the decision, similar to what we've done for the others?

Reply:
Never mind - I see now.

Reply:
But we could be a bit more flexible and call the helper 'aws_app' and switch it's behaviour.